lua/cli_flash.lua

```diff
@@ -39,6 +39,7 @@ testcli     [p][t][o] dev                      Test cli flasher
 list_interfaces[t][o]                          List all usable interfaces
 detect_netx [p][t][o]                          Detect the netx chip type
 reset_netx  [p][t][o]                          Reset the netx 90
+identify    [p][t][o]                          Blink SYS LED for 5 sec

```

Comment:
What did we want to call this? identify or identify_netx?

Reply:
I changed it to identify_netx in the newest commit